tests/scripts/feature_class_parallel.py

```diff
@@ -18,7 +18,7 @@ class Embedding(BaseModel):
 ds_name = "feature_class"
 ds = (
     DataChain.from_storage("gs://dvcx-datalakes/dogs-and-cats/")
-    .filter(C.path.glob("*cat*.jpg"))  # type: ignore [attr-defined]
+    .filter(C("file.path").glob("*cat*.jpg"))  # type: ignore [attr-defined]
```

Comment:
Q: why do we have these changes?

Reply:
New implementation of `from_storage()` is not creating deprecated root level file signals, instead they are nested inside feature object, by default it's called `file`.

Reply:
sound good, I think it might be used in many places still